Charon/filetypes/OpenPackagingConvention.py

```diff
@@ -406,7 +406,11 @@ def _readMetadata(self) -> None:
                 metadata_file = relationship.attrib["Target"]
                 if metadata_file not in self.zipfile.namelist():  # The metadata file is unknown to us.
                     continue
-                metadata = json.load(self.zipfile.open(metadata_file))
```

Comment:
@awhiemstra How did this work before? `zipfile.open` doesn't even return something that `json.load` could interpret...